cmd/mimir/help-all.txt.tmpl

```diff
@@ -1045,6 +1045,8 @@ Usage of ./cmd/mimir/mimir:
     	How frequently the compactor should run blocks cleanup and maintenance, as well as update the bucket index. (default 15m0s)
   -compactor.compaction-concurrency int
     	Max number of concurrent compactions running. (default 1)
+  -compactor.compaction-dir-cleanup-interval duration
+    	[experimental] How frequently to cleanup the compaction working directory. The directory is cleaned on startup and then only when this interval has elapsed since the last cleanup. Set to 0 to disable periodic cleanup. (default 30m0s)
```

Comment:
```suggestion
    	[experimental] Defines how frequently to clean up the compaction working directory. The directory is cleaned on startup and then only when this interval has elapsed since the last cleanup. Set to 0 to disable periodic cleanup. (default 30m0s)
```